app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if "wife" in person.keys():
+            if person["wife"]:
```

Comment:
Instead of checking if 'wife' is in person.keys() and then checking if person['wife'] is not None, you can simplify this by using `person.get('wife')`. This approach is recommended to avoid potential `KeyError` exceptions and makes the code cleaner and more concise.